### PR TITLE
+(feat): Add Font sub-setting + fitText

### PR DESCRIPTION
**Pull request by Frankie-B**
Working Font sub-setting and fitText included by default in the generator can you please review the code @mm-paulie or @mirkovw and merge them in.

### CODE REVIEW

**Comment by mirkovw** [comment]
Looks good to me! Great work @Frankie-B 

@mm-paulie I requested you to review, want to take a look and then we can merge?

File: generators/createDisplayUnitSet/templates/shared/css/style.css [hunk 1]
```diff
@@ -60,10 +57,10 @@ body {
...
   width: 80%;
   height: 50%;
+  font-family: OpenSans, sans-serif;
   font-size: 50px;
   transform: translate(-50%, -50%);
   text-align: center;
-  text-transform: uppercase;
 }
 
 .title {
```
Comment (mm-paulie) [inline generators/createDisplayUnitSet/templates/shared/css/style.css:66]: if you rename it .text, which I find it a quite common name, this class should be .text as well :). Maybe rename title to copy?

File: generators/createDisplayUnitSet/templates/shared/index.hbs [hunk 2]
```diff
@@ -19,7 +20,7 @@
     <div class="content fullscreen">
       <div class="title_container">
         <div class="title">
-          {{content.text}}
+          {{content.title}}
```
Comment (mm-paulie) [inline generators/createDisplayUnitSet/templates/shared/index.hbs]:
```
<div class="copy">
          {{content.copy}}
```

File: generators/createDisplayUnitSet/templates/shared/.sharedrc [hunk 3]
```diff
@@ -50,14 +40,13 @@
...
     "defaultFonts": [
       "OpenSans",
-      "OpenSansBold",
-      "OpenSansMedium"
+      "OpenSansBold"
     ],
     "defaultFontUrl": "./css/fonts.css",
     "clickTag": "https://www.google.com",
-    "text": "Welcome to this Banner!",
+    "title": "Welcome to this Banner!",
```
Comment (mm-paulie) [inline generators/createDisplayUnitSet/templates/shared/.sharedrc:47]: "copy": "Welcome to this Banner!",

File: generators/createDisplayUnitSet/templates/shared/.sharedrc [hunk 4]
```diff
@@ -50,14 +40,13 @@
     "css": "./css/style.css",
     "defaultFonts": [
       "OpenSans",
-      "OpenSansBold",
-      "OpenSansMedium"
+      "OpenSansBold"
```
Comment (mm-paulie) [inline generators/createDisplayUnitSet/templates/shared/.sharedrc:43]: Maybe only use "OpenSans" (and remove OpenSansBold) as an example font as we are going to change it anyway to another project related font?

File: generators/createDisplayUnitSet/templates/shared/.sharedrc [hunk 5]
```diff
@@ -14,29 +14,19 @@
...
-          "../shared/fonts/OpenSans-Bold.woff2"
+          "./fonts/OpenSans-Regular.woff",
+          "./fonts/OpenSans-Regular.woff2"
         ],
         "subset": {
           "glyphs": [
-            "content.copy1",
-            "content.copy2",
-            "content.copy3"
+            "content.title"
```
Comment (mm-paulie) [inline generators/createDisplayUnitSet/templates/shared/.sharedrc:22]: content.copy

File: generators/createDisplayUnitSet/templates/shared/css/style.css (hunk 1 above)
Comment (mirkovw) [reply generators/createDisplayUnitSet/templates/shared/css/style.css:66]: Actually, we've found that you can run into issues in the framework when the variable name is "text" or "copy" so a descriptive variable name was chosen here

**Comment by mirkovw** [comment]
> * only use OpenSans-Regular as example font in .sharedrc and css

Using 2 fonts is a good example - it may not be clear for some how to use multiple fonts

File: generators/createDisplayUnitSet/templates/shared/.sharedrc (hunk 4 above)
Comment (mirkovw) [reply generators/createDisplayUnitSet/templates/shared/.sharedrc:43]: Like I said earlier I think it's good to show how you load multiple fonts

File: generators/createDisplayUnitSet/templates/shared/.sharedrc (hunk 3 above)
Comment (mirkovw) [reply generators/createDisplayUnitSet/templates/shared/.sharedrc:47]: Maybe we can change the variable name to 'copy' but I think this is ok as well

File: generators/createDisplayUnitSet/templates/shared/index.hbs (hunk 2 above)
Comment (mirkovw) [reply generators/createDisplayUnitSet/templates/shared/index.hbs]: Same thing here - title or copy is ok with me

**Pull request merged**